Northstar.Client/mod/scripts/vscripts/ui/_menus.nut

```diff
@@ -0,0 +1,2004 @@
+untyped
```

Comment:
Adds new vanilla file. This file needs to be committed separately as `Respawn <respawn@northstar.tf>` to `main` first, when the PR has been accepted.


(Do NOT resolve until vanilla file has been committed separately, I will revert merge and rewrite commit history on `main` otherwise, you have been warned).

Reply:
Vanilla file has been added in 2376c5339776d8ef6cb6e0f60d8fcf5a61ef2885